Ews v2 bug fetch (#22645)
demisto/content: Packs/EWS/ReleaseNotes/2_0_2.md

#### Integrations
##### EWS v2

> - Fixed an issue with ***fetch-incidents***, which caused by different created and received datetime of the incident.

Reviewer: ```suggestion
- Fixed an issue with the ***fetch-incidents*** command where incidents were missed due to different DateTime of the incident.
```
